weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -460,3 +468,46 @@ const ModifiedDropdown: FC<ModifiedDropdownProps> = React.memo(
 );
 
 export default ModifiedDropdown;
+
+const OptionWithTooltip = ({text}: any) => {
```

Comment:
**suggestion:** could we avoid the `any` type here?

```suggestion
const OptionWithTooltip = ({text}: {text: string}) => {
```

Reply:
Adding the string type but needed to cast `as string` due to the Option type having many possible types
<img width="967" alt="Screenshot 2024-05-21 at 12 14 22 PM" src="https://github.com/wandb/weave/assets/126013019/d4e070f2-d3fa-4c3f-ae70-2c0d2801727e">